.azure-pipelines/gpu-tests.yml

```diff
@@ -60,6 +60,8 @@ jobs:
     - bash: |
         python requirements/collect_env_details.py
         python -c "import torch ; mgpu = torch.cuda.device_count() ; assert mgpu >= 2, f'GPU: {mgpu}'"
+        python requirements/check-avail-strategies.py
+        python requirements/check-avail-extras.py
```

Comment:
Even `horovod` is not found, the workflow doesn't fail here in CI:
https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/results?buildId=56115&view=logs&j=3afc50db-e620-5b81-6016-870a6976ad29&t=059bb337-5de1-5e6a-4c56-42f55a328c0e&l=37

Reply:
VERY good catch, thx, let's see if [7219995](https://github.com/PyTorchLightning/pytorch-lightning/pull/11819/commits/7219995ff038096c37f7412f3312ed1f434467a1) helps

Reply:
It seems to be working now :) https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/results?buildId=56189&view=logs&j=3afc50db-e620-5b81-6016-870a6976ad29&t=059bb337-5de1-5e6a-4c56-42f55a328c0e